packages/LSOnline/modules/auth/loginEventsHandler.js

```diff
@@ -13,10 +13,10 @@ mp.events.add({
     showLoginPanel(player);
     setTimeout(() => {
       if (!player.isLogged) {
-        player.kick('Byłeś zbyt długo w lobby!');
+        // player.kick(`Byłeś zbyt długo w lobby!`); // Kick == disconnect so for now we just commenting this because this ruins our playerQuit handler - we dont have enough data.
```

Comment:
In rage.mp (for now) kick == disconnect so for i just commented this because that ruins our playerQuit handler - we dont have enough data to save anything about player from kick.